### ISSUE TITLE
[Feature Request] Syncing over iCloud (or something else which is not paid)

### ISSUE THREAD

**Issue by fraander** [brentsimmons/NetNewsWire#1297]
I want to be able to sync what I have read and what I have not between two computers, as of right now I cannot do this unless I pay for Feedbin (which I do not want to pay for).

**Comment by vincode-io** [comment]
This is a duplicate of #491.

**Comment by brentsimmons** [comment]
We hear you! It‘s a good idea. We have a bunch of other things to do before we can consider this, but we will consider it.
